apps/operators/external-dns.yaml

```diff
@@ -1,5 +1,16 @@
 ---
 component: external-dns
 sources:
-  - ref: understack
-    path: 'operators/external-dns'
+  - repoURL: ghcr.io/rackerlabs/charts
+    path: .
```

Comment:
The `path: .` isn't valid cause you're loading the chart below. You're not extracting an OCI image and then loading that.